plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/ModelLayerRenderer.kt

```diff
@@ -89,6 +90,10 @@ internal class ModelLayerRenderer(
     updateLocationOrBearing()
   }
 
+  private fun setModelOpacity(opacity: Float) {
```

Comment:
same as above - if it's not used in unit tests, I'd not make a function for a single call